source/access/how_to_request_more_quota.rst

```diff
@@ -0,0 +1,89 @@
+.. _more quota:
+
+##################################
+How can I request more disk quota?
+##################################
+
+If the current quota limits of your :ref:`personal storage <data location>` or
+:ref:`Virtual Organization (VO) <virtual_organization>` are not large enough to
+carry out your research project, it might be possible to increase them. This
+option depends on data storage policies of the site managing your VSC account,
+VO or Tier-1 project as well as on current capacity of the storage system.
+
+Before requesting more storage, please check carefully the
+:ref:`current data usage of your VSC account <quota>` and identify which
+file system needs a larger quota.
+
+Personal storage
+================
+
+The following are general guidelines to request more storage in the personal
+file systems of your VSC account:
+
+VSC_HOME
+  The storage of your home directory is small on purpose. If you need more
+  storage, please switch to ``VSC_DATA``.
+
+VSC_DATA
+  On VSC sites that support :ref:`Virtual Organizations (VO)
+  <virtual_organization>`, it is preferable to join a VO to get access to a
+  larger ``VSC_DATA_VO``.
+  If that is not the case or if in doubt, please contact :ref:`user support
+  <tech support VSC>`.
+
+VSC_SCRATCH
+  On VSC sites that support :ref:`Virtual Organizations (VO)
+  <virtual_organization>`, it is preferable to join a VO to get access to a
+  larger ``VSC_SCRATCH_VO``.
+  If that is not the case or if in doubt, please contact :ref:`user support
+  <tech support VSC>`.
+
```

Comment:
Would it be useful to have site-specific links here? We have a specific page in our service catalog, personally I would prefer our users to look there.

Reply:
Sure, I'll add your link. Do you have a scpecific URL for this? or should I link to https://icts.kuleuven.be/sc/HPC?

Reply:
If possible, link to https://icts.kuleuven.be/sc/onderzoeksgegevens/HPC-storage mentioning this is specific for KU Leuven users.

Reply:
while you're at it, can you also add a link to https://hpc.vub.be/docs/vo/#requesting-more-storage-space for VUB users?